lib/v2/fzdm/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'fffdm.com': {
+        _name: '风之动漫',
+        manhua: [
+            {
+                title: '风之动漫',
+                docs: 'https://docs.rsshub.app/anime.html#feng-zhi-dong-man-feng-zhi-dong-man',
+                source: ['/*'],
+                target: (params, url) => `/fzdm/manhua/${new URL(url).pathname.split('/')[1]}/`,
```

Comment:
```suggestion
                source: ['/:id'],
                target: '/fzdm/manhua/:id',
```

Reply:
This change can't play in content page,just like `https://manhua.fffdm.com/2/1072/`

Reply:
You can use
```js
source: ['/:id', '/:id/*'],
target: '/fzdm/manhua/:id',
```
instead

Reply:
thx